docs/make.jl

```diff
@@ -1,15 +1,15 @@
-using Elysivm
+using LeastSquaresSVM
 using Documenter
 
 # * Build the complete documentation
 makedocs(;
-    modules=[Elysivm],
+    modules=[LeastSquaresSVM],
     authors="Edwin Bedolla",
-    repo="https://github.com/edwinb-ai/Elysivm/blob/{commit}{path}#L{line}",
-    sitename="Elysivm",
+    repo="https://github.com/edwinb-ai/LeastSquaresSVM/blob/{commit}{path}#L{line}",
```

Comment:
I'll be working on this, it seems that the documentation is not deploying correctly. It builds successfully, though.

Reply:
I'm glad to contribute ! :smile:

Reply:
Okay, I know what the problem is. The deployment keys do not resolve correctly due to the name change. I'll merge this and release a new hotfix of some sort.